cypress.config.js

```diff
@@ -1,21 +1,35 @@
-const { defineConfig } = require("cypress");
-const faker = require('faker');
+const { defineConfig } = require('cypress');
+const { faker } = require('@faker-js/faker');
 
 module.exports = defineConfig({
   e2e: {
     baseUrl: 'https://conduit.mate.academy/',
     setupNodeEvents(on, config) {
       on('task', {
         generateUser() {
-          const email = faker.internet.email();
+          const email = faker.internet.exampleEmail();
           const randomNumber = Math.floor(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + randomNumber,
+            username: faker.person.firstName() + randomNumber,
             email: email.toLowerCase(),
             password: '12345Qwert!'
```

Comment:
The password you're generating here is a static string. It would be a good idea to generate a unique password for each user to make the tests more robust. You can use `faker` to generate a random string.